### PR TITLE
[Feature] I added a feature to send message in dicored channel onClick send btn in contact page

### PR DESCRIPTION
**Pull request by Abdellah-Itrkine**
…k in arsil button in contact page

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,7 @@ NEXT_PUBLIC_FIREBASE_PROJECT_ID=
 NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=
 NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
 NEXT_PUBLIC_FIREBASE_APP_ID=
-NEXT_PUBLIC_MAP_BOX_ACCESS_TOKEN=
 MEASUREMENT_ID=
+NEXT_PUBLIC_MAP_BOX_ACCESS_TOKEN=
```
Comment (soufyanelf) [inline .env.example:8]: please add "DISCORD_WEB_HOOK_URL=" here

File: src/app/contact/SendMsgToDiscordChannel.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const discordWebhookUrl = process.env.DISCORD_WEB_HOOK_URL;
+
+const sendToDiscord = async (formData) => {
```
Comment (soufyanelf) [inline src/app/contact/SendMsgToDiscordChannel.jsx:3]: we have this function in 2 places

File: src/app/contact/utlis/discordUtils.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { post } from 'axios';
+
+const sendToDiscord = async (formData) => {
```
Comment (soufyanelf) [inline src/app/contact/utlis/discordUtils.js:3]: we have this function in 2 places

File: src/app/helps/[id]/ShareToSocialMediaButton/index.jsx
Comment (Abdellah-Itrkine) [inline src/app/helps/[id]/ShareToSocialMediaButton/index.jsx:1]: I deleted an old file the share button still working.

File: .env.example [hunk 4]
```diff
@@ -4,6 +4,8 @@ NEXT_PUBLIC_FIREBASE_PROJECT_ID=
 NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=
 NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
 NEXT_PUBLIC_FIREBASE_APP_ID=
-NEXT_PUBLIC_MAP_BOX_ACCESS_TOKEN=
 MEASUREMENT_ID=
-CURRENT_ENV=LOCAL
+NEXT_PUBLIC_MAP_BOX_ACCESS_TOKEN=
+DISCORD_WEB_HOOK_URL=
+NEXT_PUBLIC_CURRENT_ENV=LOCAL
```
Comment (soufyanelf) [inline .env.example:10]: Why did you update this please keeo it : CURRENT_ENV

**Pull request merged**